### ISSUE TITLE
Is prior preservation loss with class images not supported?

### ISSUE THREAD

**Issue by nayan-dhabarde** [runpod-workers/worker-dreambooth#4]
I created a custom docker image with the following script, and it stopped at 30% and worker disappears, every time.
command = [f"accelerate launch --config_file \"config.yaml\" train_dreambooth.py \
        --pretrained_model_name_or_path=\"SG161222/Realistic_Vision_V4.0\"  \
        --instance_data_dir=\"smart_cropped_images\" \
        --class_data_dir=\"man_dataset\" \
        --output_dir=\"trained_model\" \
        --train_text_encoder \
        --instance_prompt=\"a photo of nayansks man\" \
        --class_prompt=\"a photo of man\" \
        --with_prior_preservation --prior_loss_weight=1.0 \
        --resolution=512 \
        --train_batch_size=1 \
        --gradient_accumulation_steps=1 \
        --checkpointing_steps=1000 \
        --learning_rate=1e-6 \
        --lr_scheduler=\"constant\" \
        --lr_warmup_steps=0 \
        --max_train_steps=1000 \
        --num_class_images=1000 "]

Couldn't get it to run yet
